common/changes/@microsoft/api-documenter/registerStylesIE-_2019-06-21-07-21.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
All these empty change files should not be needed.  Probably they can be deleted.  I think `rush change` is detecting changes in those folders because your branch is out of sync with **master**.  I just merged master into the PR branch.

Reply:
That might be it. I will try deleting these and running `rush change` again. Could it have something to do with the fact that I'm the only dev in our team using a Mac? I've had this happen before (and I recall another recent PR where this happened).